src/main/play-doc/developer/SignalingApplicationState.md

```diff
@@ -39,3 +39,9 @@ StatusService.signalStartedOrExit()
 ```
 
 After reading the following sections, you may also wish to refer to [the reference documentation for conductr-bundle-lib](https://github.com/typesafehub/conductr-bundle-lib#typesafe-conductr-bundle-library).
+
+## Application Exit Codes
+
+ConductR uses your application's exit code to determine whether it should be restarted in order to reform the cluster. Following POSIX conventions, applications with non-zero exit codes are considered failed and will be restarted, while applications exiting with exit code 0 will be viewed as having exited normally, and thus ConductR will take no action.
+
+This is particularly important for Akka applications using [Split Brain Resolver](http://doc.akka.io/docs/akka/rp-15v09p01/scala/split-brain-resolver.html) with the [registerOnMemberRemoved](http://doc.akka.io/docs/akka/rp-15v09p01/scala/split-brain-resolver.html#Strategies) callback to shut down the ActorSystem after a split brain downing decision has been made. A non-zero return code must be issued to ensure that ConductR is instructed to recover the cluster to its previous state.
```

Comment:
I'd like to make sure I get the code sample correct, as there are different implementations of this code in different parts of the Akka documentation. I'd also like to update the corresponding Akka documentation if necessary.

The SBR documentation used to have system.registerOnTermination(System.exit(-1)) until this issue was closed: https://github.com/akka/akka/issues/18245, which changed the return code to 0, which prevents ConductR from restarting the instance. 

There is also the Cluster Usage documentation, which also shows a return code of 0 being used: http://doc.akka.io/docs/akka/snapshot/java/cluster-usage.html#How_To_Cleanup_when_Member_is_Removed

@patriknw, can you could provide some advice on this?  Should we revert the Akka documentation to use a return code of 1 (or -1) in both cases or add a note to the SBR documentation to indicate that the return code is important for ConductR? I think the later is especially important, as many customers will be implementing SBR with ConductR and likely copy/pasting the registerOnMemberRemoved code block as a starting point.